src/visualization/visalizeEpistasis/convertGene2PathwayInteractions.m

```diff
@@ -0,0 +1,75 @@
+function [neg,zer,nd,pos]=convertGene2PathwayInteractions(E,epSys,uSys)
+%[neg,zer,nd,pos]=convertGene2PathwayInteractions(E,epCmpt,uComp,Cmpt1,Cmpt2)
```

Comment:
The docu and function signature do not match (The docu has 2 more arguments).
Also could you adapt the documentation to the style guide found [here](https://opencobra.github.io/cobratoolbox/stable/contributing.html#documentation-guide)